algoliasearch/client.go

```diff
@@ -160,7 +160,7 @@ func (c *Client) GenerateSecuredAPIKey(apiKey string, params Params) (key string
 // the field used to store the index name in the queries, and the strategy used
 // to perform the multiple queries.
 // The strategy can either be "none" or "stopIfEnoughMatches".
-func (c *Client) MultipleQueries(queries []map[string]interface{}, indexField, strategy string) (res []MultipleQueryRes, err error) {
+func (c *Client) MultipleQueries(queries []Params, indexField, strategy string) (res []MultipleQueryRes, err error) {
```

Comment:
I intentionnaly didn't replace those types because the paramater wasn't actually a `Params` map. Thinking again about it, don't you think it would be a good idea to rename the `Params` type to `JsonMap` for instance? It would be more generic and understandable and could be used for both `queries` or `params` parameters (sorry for the nitpicking).